sshd_config

```diff
@@ -104,6 +110,10 @@ Ciphers aes256-gcm@openssh.com,aes128-gcm@openssh.com
 HostKeyAlgorithms rsa-sha2-512,rsa-sha2-256,ssh-ed25519
 MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com
 
+# only use hostkeys with secure algorithms, and omit the ones using NIST curves
+HostKey /etc/ssh/ssh_host_ed25519_key
+HostKey /etc/ssh/ssh_host_rsa_key
```

Comment:
```suggestion
```

Reply:
It should be fine only to enable the ssh_host_ed25519_key

Refer to: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_basic_system_settings/assembly_using-secure-communications-between-two-systems-with-openssh_configuring-basic-system-settings#making-openssh-more-secure_assembly_using-secure-communications-between-two-systems-with-openssh